lib/tasks/change-collection-wait-for-sync.js

```diff
@@ -0,0 +1,93 @@
+/* jshint globalstrict:false, strict:false, sub: true */
+/* global print, db */
+exports.name = "change-collection-wait-for-sync";
+exports.group = "inspect tasks";
+exports.args = [
+  {
+    name: "database",
+    optional: false,
+    type: "string",
+    description: "a single database, a list of databases seperated by comma or '*'"
+  },
+  {
+    name: "waitForSync",
+    optional: false,
+    type: "boolean",
+    description: "new value for 'waitForSync'"
+  },
+  {
+    name: "include-system",
+    optional: true,
+    type: "boolean",
+    description: "include system collections as well; default is false"
+  }
+];
+exports.args_arangosh = " --server.endpoint AGENT-OR-COORDINATOR";
+exports.description = "Changes the waitForSync in all collections";
+exports.selfTests = ["arango", "db", "coordinatorConnection"];
+exports.requires = "3.3.23 - 3.8.99";
```

Comment:
did we have wait for sync in 3.3?

Reply:
yes, that has been there from the beginning